Aliases/map - move - over/5eb8d24b-d660-42e5-bbea-47a659bb0827 - Main GVAR.gvar

```diff
@@ -1128,6 +1146,69 @@ if args.last('tokenimport'):
  desc.append(f"Token shortcode for `{args.last('tokenimport')}` can be found at [This Link](https://token.otfbm.io/meta/{tokenoutput}){newline}{newline}You can attach that shortcode to a target with `-token [shortcode]`")
 </drac2>
 
+<drac2>
+if args.last('savebattle') or args.last('deletebattle') or args.last('loadbattle'):
+ userBattles = load_json(get('mapBattles', "{}")) 
+ if args.last('savebattle'):
+  argsBattle = args.last('savebattle')
+  mapBGState = {}
+  for key, value in mapinfo.items():
+   if value:
+    if mapinfo.get('background')!= None:
+     mapBGState["mapbg"] = mapinfo.get('background')
+    mapBGState["mapsize"] = mapsize
+    if mapinfo.get('options')!= None:
+     mapBGState["mapoptions"] = mapinfo.get('options')
+    if mapinfo.get('walls')!= None:
+     mapBGState["walls"] = mapinfo.get('walls').split(', ')
+    if mapinfo.get('objects')!= None:
+     mapBGState["objects"] = mapinfo.get('objects').split('/')
+    break 
+  savedBGState = {"mapattach":mapBGState}
+  savedBGState.update(out)
+  savedBattle = {argsBattle:savedBGState}
+  userBattles.update(savedBattle)
+  set_uvar("mapBattles",dump_json(userBattles))
+  desc.append(f"Saved the battle `{argsBattle.title()}`")
```

Comment:
I'm partial to not doing a `.title()` here, as it won't match what the actually name it. I know we do a non-case-sensitive search, but it feels weird calling it something here and something else in the uvar.

Reply:
So that was just for aesthetics. We can just use .lower() for everything

Reply:
No, don't use `.lower()`, literally just have it be `desc.append(f"Saved the battle `{argsBattle}`")`